week1/hello/hello.c

```diff
@@ -0,0 +1,8 @@
+#include <stdio.h>
+#include <cs50.h>
+
+int main(void)
+{
+    string name = get_string("What´s your name? ");
+    printf("Hello, %s\n", name);
+}
```

Comment:
```suggestion
}

```

Always add a new line at the end of the file.